### ISSUE TITLE
OPML import: Auto update feed description

### ISSUE THREAD

**Issue by Wanabo** [FreshRSS/FreshRSS#1223]
Descriptions of the feeds are empty when you import through OPML. Not sure if descriptions are empty when you manually add a feed.

I would request an extra option to check for feed descriptions once during import or as an extra button to retrieve the description manually. If a feed has a description then no auto update of the description should take place.

**Comment by Alkarex** [comment]
Could you please provide an example of OPML file containing descriptions, which are not imported in FreshRSS?

**Comment by Alkarex** [comment]
Yes, the OPML standard support descriptions http://dev.opml.org/spec2.html
I have not double-checked, but FreshRSS should import these descriptions, and export them as well.
When you manually add a feed in FreshRSS, the description is automatically imported.

So the only problem I can see is if you end up with an OPML file in which descriptions have not been included.

**Comment by Wanabo** [comment]
Ok it seems to be a valid problem only for (empty descriptions|not exported in OPML).
He he finally we narrow it down to the real problem.

**Comment by Wanabo** [comment]
Don't know if that is something worth to take that in account. On the other hand it will make FreshRSS very userfriendly.

**Comment by Alkarex** [comment]
Related: https://github.com/FreshRSS/FreshRSS/issues/1042

**Comment by math-GH** [comment]
Is it still an open feature request?

**Comment by polybjorn** [comment]
I think this one can be closed. #3071 already handles it: empty name, website, and description get filled in from the live feed on the first refresh after OPML import, and non-empty values are left alone. Wanabo's other ask (a way to fetch a description manually) is just the per-feed reload button, which runs the same logic.

The only thing left is the wait between import and the first cron refresh, up to the configured interval (recommended 13,43, so 30 min worst case). Doing it during the import itself would mean N sequential HTTP fetches behind one request, which isn't a good trade.
